test2.js

```
@@ -0,0 +1,20 @@
+var d = [1,
+    2];
+var e = [function foo() {
+    dosomething();
+}];
+
+var foo = { "bar": "This is a bar.", "baz": { "qux": "This is a qux" }, "difficult": "to read" };
+
+typeof foo === "strnig"
+
+
+if (!key in object) {
```

Biome

[error] The negation operator is used unsafely on the left side of this binary expression.

Unsafe fix: Wrap the expression with a parenthesis


(lint/suspicious/noUnsafeNegation)